content/src/service/snapshots/SnapshotManager.ts

```diff
@@ -78,7 +78,7 @@ export class SnapshotManager {
           const buffer = Buffer.from(JSON.stringify(inArrayFormat))
 
           // Calculate the snapshot's hash
-          const hash = await Hashing.calculateBufferHash(buffer)
+          const hash = await Hashing.calculateIPFSHash(buffer)
```

Comment:
Why are you sure that all the entities are v4?

Reply:
This has nothing to do with the entities themselves. The snapshot file will be exposed with the new hash, that's it. It doesn't matter if the entities are v3 or v4